src/App/DeliveryOptions/Service/DeliveryOptionsService.php

```diff
@@ -93,14 +94,20 @@ public function createAllCarrierSettings(PdkCart $cart): array
 
         [$packageType, $carriers] = $this->getValidCarrierOptions($cart);
 
+        $showPriceSurcharge =
```

Comment:
Dit hoef je niet als parameter mee te geven, dit is gecached dus je mag het "gratis" opnieuw ophalen. Dat is ook netter dan zo'n chain waar je 1 parameter voor 1 specifiek doeleinde steeds doorgeeft.